Makefile

```
@@ -71,7 +71,11 @@ check:
 		$(CHECK_HASH_CMD) > .check.sha256; \
 	fi
 
-_run-checks: analyze test locale-check fix fmt
+_run-checks: policy-check analyze test locale-check fix fmt
```

checkmake (0.3.2)

[warning] Target "_run-checks" should be declared PHONY.

(phonydeclared)